lumeer-storage/lumeer-storage-api/src/main/java/io/lumeer/storage/api/dao/CollectionDao.java

```diff
@@ -43,6 +43,8 @@
 
    Collection getCollectionByCode(String code);
 
+   List<Collection> getAllCollections();
```

Comment:
We do not need this method. There is rarely a case when you really want to get all collections. We should always use `getCollections(SearchQuery query)` method with the right pagination.